cmake/Dependencies.cmake

```diff
@@ -60,6 +61,13 @@ if(USE_CUDA)
     else()
       caffe2_update_option(USE_CUSPARSELT OFF)
     endif()
+    if(CAFFE2_USE_CUFILE)
+      list(APPEND Caffe2_CUDA_DEPENDENCY_LIBS torch::cufile)
+    else()
+      if(NOT DEFINED ENV{TH_BINARY_BUILD} AND NOT WIN32)
```

Comment:
without this if statement I had some errors with USE_CUFILE not being in CACHE, but not sure whether this is right

Reply:
I would not expect this to be necessary given that no other entry here needs it.
I guess you made sure to make clean to remove any cached file locally?

Reply:
Specifically I got the cache related errors on windows and manywheel (not locally)